syntax = "proto3";
// minigame 客户端之前有一个 c# 写的服务器，已经对接好了，这里为了配合客户端，格式完全按照原来的做
message C2R_Login // IRequest
{
    int32 RpcId = 90;
    string Account = 1; // 帐号
    string Password = 2; // 密码
}

message R2C_Login // IResponse
{
    int32 RpcId = 90;
    int32 Error = 91;
    string Message = 92;
    string Address = 1;
    int64 Key = 2;
}